src/config/rollup.config.js

```diff
@@ -116,5 +145,24 @@ module.exports = {
     }),
     replace(replacements),
     minify ? uglify() : null,
+    codeSplitting &&
+      ((writes = 0) => ({
```

Comment:
Rather than inlining this function, could we extract it from here and give it a name?

Reply:
Sure, let's just agree on everything else first (dont want to refactor this one in case the overall feature has to be refactored)